src/module.c

```diff
@@ -1300,6 +1320,26 @@ int RM_BlockedClientMeasureTimeEnd(RedisModuleBlockedClient *bc) {
     return REDISMODULE_OK;
 }
 
+/* This API allows modules to let Redis process some commands during long
+ * blocking execution of a module command. The module can call this API
+ * periodically, and after the time defined by the `script-time-limit` config,
```

Comment:
I don't understand the coupling with `script-time-limit`, can't the module decide how it's yielding to the main thread. It seems like there is a couple of different types, so maybe we can add some type of flags to the RM_Yield call.

Reply:
i rather not let the module decide that. and indeed the name of the config is a problem.
maybe we can add an alias for it.
or actually, it already has an alias (was `lua-time-limit` in the past, and now it's `script-time-limit`).
maybe we can find a better name that can cover both modules and scripts?
how about `busy_operation_threshold`?

Reply:
@oranagra It makes sense. maybe `busy-reply-threshold`?